serviceWrappers/linux/kLiveController

```diff
@@ -32,18 +37,18 @@ HOME=${HOME:-~}
 
 loadNvm() {
     # This loads nvm
-    cd $KLIVE_CONTROLLER_PREFIX
-    [ -z "$NVM_DIR" ] && NVM_DIR="$HOME/.nvm"
-    if [ -s "$NVM_DIR/nvm.sh" ] ; then
-        source "$NVM_DIR/nvm.sh"
+    cd ${KLIVE_CONTROLLER_PREFIX}
```

Comment:
Why is cd needed here? if you just need to check that ${KLIVE_CONTROLLER_PREFIX}/.nvmrc isn't empty then use the full path and check.